### ISSUE TITLE
pptx writer creates corrupted output if document title block is used in rst

### ISSUE THREAD

**Issue by goekce** [jgm/pandoc#4181]
Using pandoc 2.0.5

I generate beamer and pptx output using rst:

File test.rst:
```
****************
 Document Title
****************

:Author: author
:Date: date

Section 1
---------

Slide 1
=======

content 1
```

Using
`
pandoc --to beamer test.rst -o test.pdf
`
generates a PDF file with three slides: document title slide including author and date, section slide and one content slide.

Using
`
pandoc --to pptx test.rst -o test.pptx
`
generates a corrupt pptx file. The repaired pptx file shows three pages without the document title slide and wrong structure (e.g., "Section 1" is used as the title for the content slide). I am using Powerpoint 2010.

Converting after removing the document title block (first 6 lines) is successful.

**Comment by jgm** [comment]
Does it change anything if you add the -s flag?

    pandoc -s --to pptx test.rst -o test.pptx

**Comment by goekce** [comment]
`pandoc -s --to pptx test.rst -o test.pptx`
yields
`Could not find data file /usr/share/pandoc/data/templates/default.pptx`

**Comment by jkr** [comment]
I'll take a look

**Comment by jkr** [comment]
Two issues here:

1. Related to the lack of defiition list implementation (that's how rst title blocks work). I've been meaning to implement that, and will do that now to see how it affects the issue.

2. Issue with standalone / lack of default. I should put that default.pptx in, but I'm afraid that it will encourage use of reference-doc which doesn't work very well. Since pptx is always standalone, I'll look into the best way to disable this for now.

**Comment by jkr** [comment]
Sorry -- now I see the issue further. RST reader behaves differently in standalone mode. So best way to deal with this at the moment is fix #2 above and make sure the pptx writer insists on standalone.
